client/src/shared/components/video/VideoSideMenuItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import type { VideoListData } from "@/domain/student/types/video";
+import VideoSideMenuChapterHeader from "@/shared/components/video/VideoSideMenuChapterHeader";
+import VideoSideMenuVideoList from "@/shared/components/video/VideoSideMenuVideoList";
+
+interface VideoSideMenuItemProps {
+  videoLists: VideoListData[];
+  activeVideoId?: number;
+  onSelectVideo?: (videoId: number) => void;
+  isMenuOpen: boolean;
+}
+
+const VideoSideMenuItem = ({
+  videoLists,
+  activeVideoId,
+  onSelectVideo,
+  isMenuOpen,
+}: VideoSideMenuItemProps) => {
+  const [openMap, setOpenMap] = useState<Record<number, boolean>>(() => {
+    const init: Record<number, boolean> = {};
+    videoLists.forEach((list) => {
+      if (list.contentsId) init[list.contentsId] = true;
+    });
+    return init;
+  });
```

Comment:
p3: reduce를 쓸 수도 있을 것 같아요!

```suggestion
const [openMap, setOpenMap] = useState<Record<number, boolean>>(
  () =>
    videoLists.reduce((acc, list) => {
      if (list.contentsId) acc[list.contentsId] = true;
      return acc;
    }, {} as Record<number, boolean>)
);
```